Gems/Atom/Feature/Common/Code/Source/Material/MaterialConverterSystemComponent.cpp

```diff
@@ -158,6 +158,10 @@ namespace AZ
             }
 
             sourceData.SetPropertyValue(Name{"opacity.factor"}, materialData.GetOpacity());
+            if (abs(materialData.GetOpacity() - 1.0f) > AZ::Constants::FloatEpsilon)
```

Comment:
Why not just use 1.0 - opacity?

Reply:
the value of opacity between 0 to 1.0f, its ok to use `1.0 - opacity`.